File: redcap_v15.7.6/API/examples/r/import_user_dag_maps.R
#!/usr/bin/env Rscript

source('config.R')
library(RCurl)
library(jsonlite)

record <- c(
	username='testuser',
	redcap_data_access_group='api_testing_group'
)

data <- toJSON(list(as.list(record)), auto_unbox=TRUE)

result <- postForm(
    api_url,
    token=api_token,
    content='userDagMapping',
	action='import',
    format='json',
    data=data
)
print(result)

File: redcap_v15.7.6/API/examples/python/import_user_dag_maps.py
#!/usr/bin/env python

from config import config
import requests, json

record = {
    'username': 'testuser',
    'redcap_data_access_group': 'api_testing_group'
}

data = json.dumps([record])

fields = {
    'token': config['api_token'],
    'content': 'userDagMapping',
    'action': 'import',
    'format': 'json',
    'data': data,
}

r = requests.post(config['api_url'],data=fields)
print('HTTP Status: ' + str(r.status_code))
print(r.text)